scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -118,6 +116,45 @@ public static BuildTargetResult PublishDebFileToDebianRepo(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target]
+        [Environment("DOCKER_HUB_REPO")]
+        [Environment("DOCKER_HUB_TRIGGER_TOKEN")]
+        public static BuildTargetResult TriggerDockerHubBuilds(BuildTargetContext c)
+        {
+            string dockerHubRepo = Environment.GetEnvironmentVariable("DOCKER_HUB_REPO");
+            string dockerHubTriggerToken = Environment.GetEnvironmentVariable("DOCKER_HUB_TRIGGER_TOKEN");
+
+            Uri baseDockerHubUri = new Uri("https://registry.hub.docker.com/u/");
+            Uri dockerHubTriggerUri;
+            if (!Uri.TryCreate(baseDockerHubUri, $"{dockerHubRepo}/trigger/{dockerHubTriggerToken}/", out dockerHubTriggerUri))
+            {
+                return c.Failed("Invalid DOCKER_HUB_REPO and/or DOCKER_HUB_TRIGGER_TOKEN");
+            }
+
+            c.Info("Triggering automated DockerHub builds for " + dockerHubRepo);
```

Comment:
nit - This is another place to consider using the $ interpolated string syntax.  This would make the code more consistent.